include/linux/mroute.h

```diff
@@ -2,10 +2,8 @@
 #ifndef _UAPI__LINUX_MROUTE_H
 #define _UAPI__LINUX_MROUTE_H
 
-#ifdef __LINUX__
```

Comment:
This is going to break *bsd* code right?  This is a no go from my perspective.

The code in include/linux/mroute.h should only be copies of the system headers and we should not be modifying them.  Why is __LINUX__ not a symbol on Alpine?